### PR TITLE
[FIX] Empty buffer on PO line when receiving in 2 steps

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by max3903** [comment]
@guewen, @jbeficent, @lreficent, @jgrandguillaume If you have a better way to solve this, I am open to suggestions.

File: ddmrp/models/purchase_order.py [hunk 1]
```diff
@@ -19,6 +19,23 @@ def create(self, vals):
         record._calc_execution_priority()
         return record
 
+    @api.multi
+    @api.depends("product_id")
+    def _compute_orderpoint_id(self):
+        for rec in self:
+            if rec.order_id.origin and not rec.orderpoint_id:
```
Comment (LoisRForgeFlow) [inline ddmrp/models/purchase_order.py]: you can prefilter above the self with this condition

File: ddmrp/models/purchase_order.py [hunk 2]
```diff
@@ -19,6 +19,23 @@ def create(self, vals):
...
+                group_name = rec.order_id.origin.split(", ")[-1]
+                if group_name != '':
+                    group_id = self.env['procurement.group'].\
+                        search([('name', '=', group_name)])
+                    if group_id:
+                        orderpoint_id = \
+                            self.env['stock.warehouse.orderpoint'].\
+                            search([('product_id', '=', rec.product_id.id),
+                                    ('group_id', '=', group_id.id)])
```
Comment (LoisRForgeFlow) [inline ddmrp/models/purchase_order.py]: limit=1?

**Review by LoisRForgeFlow — COMMENTED** [review]
If you add a test for this it would be great

**Comment by JordiBForgeFlow** [comment]
I think that the approach of this PR is not the right one. You should define a separate module in OCA/stock-logistics-workflow called 'stock_orderpoint_purchase_link'.

Then define in purchase.order.line a field 'orderpoint_ids' as many2many.
Then pass on the orderpoint from the procurement rule.
Equivalent to what we did with stock requests here: 
* https://github.com/OCA/stock-logistics-warehouse/blob/11.0/stock_request_purchase/models/procurement_rule.py
* https://github.com/OCA/stock-logistics-warehouse/blob/11.0/stock_request_purchase/models/purchase_order_line.py#L11

**Comment by JordiBForgeFlow** [comment]
Then, add to the ddmrp this new module as dependency, and benefit from the orderpoint_ids now available in the PO line to obtain the execution_priority_level

**Comment by LoisRForgeFlow** [comment]
I have adapted this as per the new approach see https://github.com/Eficent/ddmrp/pull/63

**Pull request closed without merging**